File: modulo1/segmento3/apunte1/Ejercicio4.java
import java.util.Scanner;

public class Ejercicio4 {
	public static void main(String args[]){
		Scanner teclado = new Scanner(System.in);

		System.out.print("Ingrese un número: ");
		Integer number1 = Integer.parseInt(teclado.nextLine());

		System.out.print("Ingrese otro número: ");
		Integer number2 = Integer.parseInt(teclado.nextLine());

		System.out.print("Ingrese otro número: ");
		Integer number3 = Integer.parseInt(teclado.nextLine());

		System.out.print("Ingrese otro número: ");
		Integer number4 = Integer.parseInt(teclado.nextLine());

		if(number1 == number2)
			System.out.println(number1.toString() + " y " + number2.toString() +							 " son iguales");

		if(number3 == number4)
			System.out.println(number3.toString() + " y " + number4.toString() +							 " son iguales");
	}
}

File: modulo1/segmento3/apunte1/ejercicio4.web/ejercicio4.js
function sameNumbers(){
	const number1 = parseInt(prompt('Ingrese un numero: '));
	const number2 = parseInt(prompt('Ingrese otro numero: '));
	const number3 = parseInt(prompt('Ingrese otro numero: '));
	const number4 = parseInt(prompt('Ingrese otro numero: '));

	if(number1 == number2)
		alert(number1 + ' y ' + number2 + ' son iguales');

	if(number3 == number4)
		alert(number3 + ' y ' + number4 + 'son iguales');
}
